shell/common/common_TPC-H.sh

```diff
@@ -116,31 +113,31 @@ tpc-h_cmd_datagen() {
-  hadoop_delete_path "$bench_name" "$TPCH_HDFS_DIR/$TPCH_SCALE_FACTOR"
-  execute_hadoop_new "$bench_name" "fs -mkdir -p $TPCH_HDFS_DIR/$TPCH_SCALE_FACTOR/{customer,lineitem,nation,orders,part,partsupp,region,supplier}"
-
-  execute_hadoop_new "$bench_name" "fs -moveFromLocal $D2F_local_dir/tpch/tpch-gen/target/tools/customer.tbl $TPCH_HDFS_DIR/$TPCH_SCALE_FACTOR/customer/"
-  execute_hadoop_new "$bench_name" "fs -moveFromLocal $D2F_local_dir/tpch/tpch-gen/target/tools/lineitem.tbl $TPCH_HDFS_DIR/$TPCH_SCALE_FACTOR/lineitem/"
-  execute_hadoop_new "$bench_name" "fs -moveFromLocal $D2F_local_dir/tpch/tpch-gen/target/tools/nation.tbl $TPCH_HDFS_DIR/$TPCH_SCALE_FACTOR/nation/"
-  execute_hadoop_new "$bench_name" "fs -moveFromLocal $D2F_local_dir/tpch/tpch-gen/target/tools/orders.tbl $TPCH_HDFS_DIR/$TPCH_SCALE_FACTOR/orders/"
-  execute_hadoop_new "$bench_name" "fs -moveFromLocal $D2F_local_dir/tpch/tpch-gen/target/tools/part.tbl $TPCH_HDFS_DIR/$TPCH_SCALE_FACTOR/part/"
-  execute_hadoop_new "$bench_name" "fs -moveFromLocal $D2F_local_dir/tpch/tpch-gen/target/tools/partsupp.tbl $TPCH_HDFS_DIR/$TPCH_SCALE_FACTOR/partsupp/"
-  execute_hadoop_new "$bench_name" "fs -moveFromLocal $D2F_local_dir/tpch/tpch-gen/target/tools/region.tbl $TPCH_HDFS_DIR/$TPCH_SCALE_FACTOR/region/"
-  execute_hadoop_new "$bench_name" "fs -moveFromLocal $D2F_local_dir/tpch/tpch-gen/target/tools/supplier.tbl $TPCH_HDFS_DIR/$TPCH_SCALE_FACTOR/supplier/"
+  hadoop_delete_path "$bench_name" "$TPCH_HDFS_DIR/$(get_benchmark_data_size_gb)"
+  execute_hadoop_new "$bench_name" "fs -mkdir -p $TPCH_HDFS_DIR/$(get_benchmark_data_size_gb)/{customer,lineitem,nation,orders,part,partsupp,region,supplier}"
+
+  execute_hadoop_new "$bench_name" "fs -moveFromLocal $D2F_local_dir/tpch/tpch-gen/target/tools/customer.tbl $TPCH_HDFS_DIR/$(get_benchmark_data_size_gb)/customer/"
+  execute_hadoop_new "$bench_name" "fs -moveFromLocal $D2F_local_dir/tpch/tpch-gen/target/tools/lineitem.tbl $TPCH_HDFS_DIR/$(get_benchmark_data_size_gb)/lineitem/"
+  execute_hadoop_new "$bench_name" "fs -moveFromLocal $D2F_local_dir/tpch/tpch-gen/target/tools/nation.tbl $TPCH_HDFS_DIR/$(get_benchmark_data_size_gb)/nation/"
+  execute_hadoop_new "$bench_name" "fs -moveFromLocal $D2F_local_dir/tpch/tpch-gen/target/tools/orders.tbl $TPCH_HDFS_DIR/$(get_benchmark_data_size_gb)/orders/"
+  execute_hadoop_new "$bench_name" "fs -moveFromLocal $D2F_local_dir/tpch/tpch-gen/target/tools/part.tbl $TPCH_HDFS_DIR/$(get_benchmark_data_size_gb)/part/"
+  execute_hadoop_new "$bench_name" "fs -moveFromLocal $D2F_local_dir/tpch/tpch-gen/target/tools/partsupp.tbl $TPCH_HDFS_DIR/$(get_benchmark_data_size_gb)/partsupp/"
+  execute_hadoop_new "$bench_name" "fs -moveFromLocal $D2F_local_dir/tpch/tpch-gen/target/tools/region.tbl $TPCH_HDFS_DIR/$(get_benchmark_data_size_gb)/region/"
+  execute_hadoop_new "$bench_name" "fs -moveFromLocal $D2F_local_dir/tpch/tpch-gen/target/tools/supplier.tbl $TPCH_HDFS_DIR/$(get_benchmark_data_size_gb)/supplier/"
```

Comment:
Since we're reusing the value so many times in this function, it would probably make sense to define a local at the top, e.g.:
```
local src_base_dir=$D2F_local_dir/tpch
local dst_base_dir=$TPCH_HDFS_DIR/$(get_benchmark_data_size_gb)
```
Since you're changing these lines, you might as well put quotes in the right places too:
```
execute_hadoop_new "$bench_name" "fs -moveFromLocal \"$src_base_dir/tpch-gen/target/tools/supplier.tbl\" \"$dst_base_dir/supplier/\""
```
```